### PR TITLE
Add Installation Guide for Maven and Gradle Project

### PR DESCRIPTION
**Pull request by kevindavee**
Resolves: #31

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -84,18 +84,64 @@ Please check [Xendit API Reference](https://xendit.github.io/apireference/).
 JDK 1.7 or later.
 
 ## Installation
-Maven:
+####Maven
+
+Add this line of code in your `pom.xml`
```
Comment (stanleynguyen) [inline README.md]: these lines?

File: README.md [hunk 2]
```diff
@@ -84,18 +84,64 @@ Please check [Xendit API Reference](https://xendit.github.io/apireference/).
...
+    </name>
+    <url>
+      https://dl.bintray.com/xendit/android
+    </url>
+  </pluginRepository>
+</pluginRepositories>
+```
+
+####Gradle
+Add this line in your gradle build
```
Comment (stanleynguyen) [inline README.md]: same as above. these lines

Comment (kevindavee) [reply README.md]: Fixed

File: README.md (hunk 1 above)
Comment (kevindavee) [reply README.md]: Fixed

File: README.md [hunk 3]
```diff
@@ -84,18 +84,65 @@ Please check [Xendit API Reference](https://xendit.github.io/apireference/).
...
+```
+
+####Gradle
+
+Add these lines in your `build.gradle`
+```
+...
+repositories {
+	maven {
+		url  "https://dl.bintray.com/xendit/android"
```
Comment (tohyongcheng) [inline README.md]: why is this android?

Comment (kevindavee) [reply README.md]: It is a legacy repository. We will fix this after we publish the version 2.0 under the `xendit-java` name

**Pull request merged**